.github/workflows/on_merge_to_main_cypress.yml

```diff
@@ -0,0 +1,94 @@
+name: Merge to main Cypress for VRT Baseline
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # run this job first to create a single timestamped id for all percy batches on parallel machines
+  percy-batch-id:
```

Comment:
question: why didn't we add all this to `on_merge_to_main.yml`? Is it because of this percy batch id job?

Reply:
Hmmm I think I naively followed the pattern from the existing cypress workflow, but in hindsight that's only done because of the events different from `on_pull_request`.

Probs will put it in `on_merge_to_main`